File: Python Word Examples/SectionsApiDemo/addSection.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.SectionsApi(configuration)

file_name = 'Template.docx'
extra = { 'folder': "input", 'section_index': 2,'dest_file_path': 'output/AddSection.docx'}
result =api.add_section(file_name, **extra)

File: JavaScript Word Examples/SectionsApiDemo/addSection.js
(function (){
    var appId = "your id";
    var appKey = "your key";      
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.SectionsApi(apiClient);

    var destFilePath = 'output/AddSection.docx';
    var opt = { folder: 'input' };
    instance.addSection("Template.docx", destFilePath, opt,
        function (error, data, response) {
            if (error) throw error;
        });
})();